app/components/learnergroup-members.js

```diff
@@ -1,7 +1,19 @@
 import Ember from 'ember';
 import DS from 'ember-data';
+import InPlace from 'ilios/mixins/inplace';
+
+const { computed, isEmpty, ObjectProxy, observer, RSVP } = Ember;
+const { all } = RSVP;
+
+export default Ember.Component.extend(InPlace, {
+  init() {
```

Comment:
I am going to remove the `init` method for this particular case because the component is huge and has dependencies. When you have a smaller component (w/o other dependent components) that is going to be used a lot, it's good practice to use the `init` method because setting them explicitly would put them on component's prototype, which is shared across all instances.  It's just good practice.